src/commands/generate/types.ts

```diff
@@ -0,0 +1,88 @@
+import { CSharpFileGenerator, JavaFileGenerator, JavaScriptFileGenerator, TypeScriptFileGenerator, GoFileGenerator, Logger} from '@asyncapi/modelina';
+import { flags } from '@oclif/command';
+import Command from '../../base';
+import { load } from '../../models/SpecificationFile';
+import { parse } from '@asyncapi/parser';
+export default class Types extends Command {
+  static description = 'Generates typed models';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    file: flags.string({ char: 'f', description: 'path to the AsyncAPI file to generate types for' }),
+    outputDirectory: flags.string({ char: 'o', description: 'output path to write the types to', required: true}),
```

Comment:
why not `output` as it is already in generator cli? nobody complained on it

Reply:
That makes sense 👍